packages/dumb/src/components/settings/history/history-table-content.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2023, Diabeloop
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import { HistorySpannedRow } from './history-table-spanned-row'
+import { HistoryTableRow } from './history-table-row'
+import React, { type FunctionComponent } from 'react'
+import { type TimePrefs } from 'medical-domain'
+import { type IncomingRow } from '../../../models/historized-parameter.model'
+import { transformToViewModels } from './history-table.core'
+import type moment from 'moment-timezone'
+
+interface HistoryTableContentProps {
+  onSwitchToDaily: (date: moment.Moment | Date | number | null) => void
+  rows: IncomingRow[]
+  timePrefs: TimePrefs
+  length: number
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏I see that this props ends up being used in `packages/dumb/src/components/settings/history/history-table-spanned-row.tsx`. Its value is always `4` (as this is a component being used only once). Maybe, this value could be defined directly into `history-table-spanned-row.tsx` in a const such as `const NUMBER_OF_COLUMNS = 4`? If you disagree, I would  then advise to at least no get the value from `packages/dumb/src/components/settings/history/history-table.tsx` but directly from the file I am commenting on